modules/touch/src/touchinteraction.cpp

```diff
@@ -22,8 +22,11 @@
  * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
  ****************************************************************************************/
 
+#include <openspace/engine/globals.h>
 #include <modules/touch/include/touchinteraction.h>
 #include <modules/imgui/imguimodule.h>
+#include <modules/webbrowser/webbrowsermodule.h>
```

Comment:
This should be wrapped in #ifdef OPENSPACE_MODULE_WEBBROWSER_ENABLED, so that touch can be compiled without webbrowser

Reply:
Done